lib/matplotlib/axes/_axes.py

```diff
@@ -8226,6 +8227,17 @@ def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
           its only parameter and return a scalar. If None (default), 'scott'
           is used.
 
+        fillcolor : color or None; see :ref:`colors_def`
+          If provided, will set the fill color of the violin plots. The alpha
+          value of the fill is automatically set to 0.3.
+
```

Comment:
Please see https://matplotlib.org/stable/devel/coding_guide.html#new-features-and-api-changes

Basically, `.. versionadded:: 3.9` should be added for these three arguments. However, see the link above to get the indentation correct (otherwise I would have added a suggestion, but most likely it will be wrong...)

Reply:
Sure thing. Those lines have been added. I inherited the two space indent rather than the typical full tab for the descriptions because that is how the violinplot code was already structured. But all other details follow the example from the above link!

Reply:
Sorry, I just meant that I couldn't provide a proper suggestion as it is more or less impossible to get the spacing correct in the GitHub comment.

These looks good!